data/sources/blocks/e-commerce/use-of-cloud-computing-services-block-43.yaml

```diff
@@ -0,0 +1,52 @@
+id: use-of-cloud-computing-services-block-43
+questions:
+- answers:
+  - description: ''
+    id: use-of-cloud-computing-services-answer-43
+    label: ''
+    mandatory: true
+    options:
+    - label: 'Yes'
+      value: 'Yes'
+    - label: 'No'
+      value: 'No'
+    q_code: '358'
+    type: Radio
+    guidance:
+      show_guidance: Show Cloud Computing Definition
+      hide_guidance: Hide Cloud Computing Definition
+      content:
+      - title: Cloud Computing
+        description: "<em>ICT services</em> that are used <em>over the internet</em>\
+          \ to access software, computing power, storage capacity, etc.</br>\
+          \ Cloud computing services must have all of the following characteristics:\
+          \ <ul><li>Are delivered from <em>servers</em> of service providers</li>\
+          \ <li>Can be easily <em>scaled</em> up or down (for example; the number of users or change of storage\
+          \ capacity)</li><li>Can be used <em>on-demand by the user</em>, at least after the initial set up\
+          \ (without human interaction with the service provider)</li>\
+          \ <li>Are <em>paid</em> for, either by the user, by the capacity used or they are pre-paid</li>\
+          \ </ul>Cloud computing may include connections via Virtual Private Networks (VPN)"
```

Comment:
Lists in guidance content should be done using the list property rather than HTML tags